apps/dashboard/app/views/layouts/nav/_link.html.erb

```diff
@@ -2,13 +2,14 @@
 locals:
 title - required - text for the link
 url - required - link url string
+class - optional, main type of link.
 faicon - optional, default "cog" - font awesome icon to use
 target - optional, default "_self" - link target
 role - optional, default ""
 method - optional, data-method for the link
 %>
-<%= tag.a href: local_assigns.fetch(:url), target: local_assigns.fetch(:target, '_self'), class: "dropdown-item m-auto #{local_assigns.fetch(:role, nil)}",  'data-method': local_assigns.fetch(:method, nil) do %>
-  <%= tag.i class: "fas fa-fw fa-#{local_assigns.fetch(:faicon, 'cog')}" %>
+<%= tag.a href: local_assigns.fetch(:url), target: local_assigns.fetch(:new_tab, false) ? "_blank" : nil, class: "#{local_assigns.fetch(:class, 'nav-link')} m-auto #{local_assigns.fetch(:role, nil)}",  'data-method': local_assigns.fetch(:method, nil) do %>
+  <%= icon_tag(local_assigns.fetch(:icon_uri, URI("fas://cog"))) %>
```

Comment:
Seems like the comments need to change. We're replacing `faicon` with `icon_uri`?

Reply:
`new_tab` and `icon_uri` changes are to make them consistent with how `OodApp` and `OodAppLink` generate links and how the `_group.html.erb` renders a link too.